api-test/DEV.postman_environment.json

```diff
@@ -0,0 +1,21 @@
+{
+	"id": "4582c6a3-ae67-41b8-b577-a2c6acd27a38",
+	"name": "DEV",
+	"values": [
+		{
+			"key": "host",
+			"value": "https://api.dev.platform.pagopa.it/afm/calculator-service/v1",
```

Comment:
is this value a typo or must it reference to AFM?

Reply:
it is an error